tests/vmi_multiqueue_test.go

```diff
@@ -123,5 +124,33 @@ var _ = Describe("[Serial]MultiQueue", func() {
 			}
 		})
 
+		It("should be able to create a multi-queue VMI when requesting a single vCPU", func() {
+			vmi := libvmi.NewCirros(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
```

Comment:
I don't think that this is relevant for the test. Let's keep them to the minimum.

Reply:
Just wanted to be explicit. I understand your point though - there's nothing special about the type of interface here, so, we can use whatever kubevirt has as default.